src/console_title.go

```diff
@@ -50,6 +50,12 @@ func (t *consoleTitle) getTemplateText() string {
 	context["Path"] = t.getPwd()
 	context["Folder"] = base(t.getPwd(), t.env)
 	context["Shell"] = t.env.getShellName()
+	context["User"] = t.env.getCurrentUser()
+	if host, err := t.env.getHostName(); err == nil {
+		context["Host"] = host
+	} else {
```

Comment:
I only use if else when there's no other option. Can we initialize it as empty first and override it when there's no error?

Reply:
Done.